scss/_buttons.scss

```diff
@@ -357,31 +357,27 @@
     }
   }
 
-  .btn + .button_to,
-  .button_to + .btn,
-  .button_to + .button_to {
+  .btn + .btn-form,
+  .btn-form + .btn,
+  .btn-form + .btn-form {
```

Comment:
Technically any class renames should require a major version bump since this is not backwards *in*compatible for folks.